zipline/test_algorithms.py

```diff
@@ -947,6 +949,28 @@ def handle_data(self, data):
         self.i += 1
 
 
+class TestLiquidationAlgorithm(TradingAlgorithm):
+    def initialize(self, *args, **kwargs):
+        self.ordered = False
+        self.set_commission(PerTrade(0))
+        self.set_slippage(FixedSlippage(spread=0))
+        self.num_positions = []
+        self.cash = []
+
+    def handle_data(self, data):
+        if not self.ordered:
+            self.ordered = True
```

Comment:
General coding style note: when I have a flag signalling that some operation has been performed, I generally try to set it only after that operation has been performed.  Here it's harmless, but it makes you more likely to do the right/expected thing in more complicated setups.